File: data_structures/Linked_list/php/Node.php
<?php

class Node
{
    /**
     * @var Node
     */
    public $previous;

    /**
     * @var Node
     */
    public $next;

    /**
     * @var mixed
     */
    public $element;

    /**
     * Node constructor.
     * @param null $element
     */
    public function __construct($element = null)
    {
        $this->element = $element;
    }
}

File: data_structures/node stack/Node.java
// anode class for a singly linked list

public class Node
{
	public Object element;
	public Node next;
	
	// constructor
	public Node()
	{
		element = null;
		next = null;
	}
	
	public Node(Object e, Node n)
	{
		element = e;
		next = n;
	}
}